src/components/Header.svelte

```diff
@@ -13,6 +14,11 @@
   let dropdownOpen = false;
   let dropdownOpen2 = false;
 
+  const disconnect = () => {
+    $eth.address = null;
```

Comment:
@Alexintosh when using the resetConnection because it gives this error:

ReferenceError: connectWeb3 is not defined
    at resetWeb3Listeners (http://localhost:8080/_dist_/stores/eth/connection.js:26:33)
    at resetConnection (http://localhost:8080/_dist_/stores/eth/connection.js:93:3)
    at HTMLAnchorElement.disconnect (http://localhost:8080/_dist_/components/Header.svelte.js?mtime=1626169306070:1008:3)

Reply:
@Alexintosh let me know if resetConnection is really needed.
I've done all the possible tests without using this function, and it perfectly logout anytime.
If that's fine for you, we can consider this to be fixed, otherwise let me know how you wish me to proceed.